src/index.d.ts

```diff
@@ -82,6 +82,7 @@ export interface OnProgressEvent {
 
 export interface FastImageProperties {
     source: FastImageSource | number
+    placeholder?: React.ReactNode
```

Comment:
Maybe you can allow `number` here too? So that one could write:
```
import myPlaceholderPicture from 'src/images/myPlaceholderPicture.png';

....
	<FastImage source={{ uri: myUri }} placeholder={myPlaceholderPicture} />
```

Instead of being forced to do this, for example:

```
import { Image } from 'react-native';
import myPlaceholderPicture from 'src/images/myPlaceholderPicture.png';

....
	<FastImage source={{ uri: myUri }} placeholder={<Image source={myPlaceHolderImage}} />} />

```

Though I am assuming that the default placeholder for an image is another image.